.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,6 @@
+// This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
 {
-	"name": "Python VehicleApp",
+	"name": "VehicleApp Python app",
```

Comment:
Would you prefer `Python vehicle ${{ repoType}} development` where `repoType` is either `app` or `sdk` ? This is a substituted string coming from a CLI sync.

This would look like this:

`Python vehicle app development`
`Python vehicle sdk development`

Reply:
Yes this sounds way better. Looking at the current name, maybe even without development at the end. But not sure what is better here since it is the name of the dev container...